ui/language_utils.js

```diff
@@ -55,12 +59,27 @@ shaka.ui.LanguageUtils = class {
       }
     };
 
-    const getCombination = (language, rolesString, label) => {
+    const getCombination = (language, rolesString, label, channelsCount) => {
+      const keys = [
+        language,
+        rolesString,
+      ];
+      if (showAudioChannelCountVariants && channelsCount != null) {
+        keys.push(channelsCount);
+      }
       if (label &&
           trackLabelFormat == shaka.ui.Overlay.TrackLabelFormat.LABEL) {
-        return language + ': ' + label + ': ' + rolesString;
+        keys.push(label);
+      }
+      return keys.join(': ');
+    };
+
+    const getChannelsCountName = (channelsCount) => {
+      let name = '';
+      if (channelsCount >= 5) {
+        name = ' ' +localization.resolve(LocIds.SURROUND);
```

Comment:
Nit: Missing whitespace between the `+` and the `localization.resolve(`

Reply:
Fixed!